engine/crates/engine-v2/src/sources/graphql/query.rs

```diff
@@ -232,9 +301,19 @@ impl std::ops::DerefMut for Buffer {
 }
 
 impl Buffer {
-    fn indent_write(&mut self, s: impl AsRef<str>) -> Result<(), std::fmt::Error> {
-        let indent = "\t".repeat(self.indent);
-        self.write_str(&indent)?;
-        self.write_str(s.as_ref())
+    fn into_string(self) -> String {
+        self.inner
+    }
+
+    fn write_indent(&mut self) {
+        let mut indent = self.indent;
+        loop {
+            if indent < INDENTS {
+                self.inner.push_str(&SPACES[..self.indent * INDENT_SIZE]);
+                return;
+            }
+            self.inner.push_str(&SPACES[..INDENTS * INDENT_SIZE]);
+            indent -= INDENTS
+        }
```

Comment:
Seems like it'd be a lot easier to just do `self.inner.push(' ')` in a loop...

Reply:
hum.... indeed...